test/test_markov_approx.jl

```diff
@@ -16,4 +16,27 @@
             @test isapprox(sum(mc.state_values), 0.0; rough_kwargs...)
         end
     end
+
+
+    #
+    # Test discrete estimation
+    #
+    P = [0.5 0.25 0.25
+         0.25 0.5 0.25
+         0.25 0.25 0.5]
+
+    # Test with numerical inputs
+    mc = MarkovChain(P, [0.0, 0.5, 1.0])
+    X = simulate(mc, 100_000)
+    mc2 = estimate_MC_discrete(X)
+    @test isapprox(mc.state_values, mc2.state_values, atol=1e-10)
+    @test isapprox(mc.p, mc2.p, atol=1e-2)
+
+    # Test with other inputs
+    mc = MarkovChain(P, ["a", "b", "c"])
+    X = simulate(mc, 100_000)
+    mc2 = estimate_MC_discrete(X)
+    @test isapprox(mc.state_values, mc2.state_values, atol=1e-10)
+    @test isapprox(mc.p, mc2.p, atol=1e-2)
```

Comment:
Good thought. This should be tested.

I had originally planned on piggy backing on the other tests in `test_markov_approx.jl`, but when `n` got very large the simulations didn't cover all of the states in Rouwenhorst method so I decided just to write some smaller examples.

Done.